project/MimaExcludes.scala

```diff
@@ -1129,6 +1129,12 @@ object MimaExcludes {
       ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("org.apache.spark.ml.param.shared.HasLoss.org$apache$spark$ml$param$shared$HasLoss$_setter_$loss_="),
       ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("org.apache.spark.ml.param.shared.HasLoss.getLoss"),
       ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("org.apache.spark.ml.param.shared.HasLoss.loss")
+    ) ++ Seq(
+      // [SPARK-9853][Core] Optimize shuffle fetch of contiguous partition IDs.
+      ProblemFilters.exclude[MissingTypesProblem]("org.apache.spark.FetchFailed$"),
```

Comment:
We can get rid of this if we don't touch `FetchFailed`.

Reply:
I see, so I will keep FetchFailed as it was (no numBlocks is introduced), is it OK?

Reply:
yup